### ISSUE TITLE
Xenial version number should be higher than vivid version number

### ISSUE THREAD

**Issue by NeoTheThird** [UbuntuOpenStore/openstore-meta#205]
TL;DR: Please publish a xenial-only click of the openstore-app that has a higher version number than the vivid version. Fixes https://github.com/ubports/ubuntu-touch/issues/725.

If a system-image update is installed without wipe and an already installed app has a higher version number than the one from the system-image, the local version will be kept. This results in problems when upgrading from vivid to xenial, since the openstore app will just stay the vivid version, which means it won't work in xenial. Could we publish a new openstore click with a higher version number only for xenial?

It might be a good idea to force xenial version numbers to be higher than vivid version numbers server-side for all apps. That way a user installing ota-4 could just go ahead and update all his apps and have the xenial versions. Also, it might make sense to allow publishing xenial-only packages from now on.

**Comment by NeoTheThird** [comment]
Sorry about that, already done. *headdesk*
